scripts/test.sh

```diff
@@ -61,12 +61,14 @@ then
   echo "     bash $0"
   echo ""
   return 1
-fi
+fi 
 
 set -e
 source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
 
+pip install numpy==1.6.1 --target="$THIRD_PARTY_DIR"
```

Comment:
Some thoughts:

- Could you add some comments above this file explaining why we're installing it here, instead of via the normal process in install_third_party.py?
- Could we install this in ./third_party/pip-packages instead? It's a different pattern from the others, so I'd like to separate it out.
- When the test script is run a second time, it should not reinstall numpy if numpy has already been installed.